docs/releases/2.20.0.md

```diff
@@ -13,6 +16,9 @@ This release contains changes that you may need to be aware of. Read carefully b
 
 ## Deprecations
 
+### Kubernetes minimum supported version is now 1.25, tested up to 1.30
+* The minimum supported version of Kubernetes is now 1.25, owing to deprecations in some core [API functions](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-25), and the availability of upstream charts. The v2.20.0 release will not install into an earlier version of Kubernetes, so any Kubernetes upgrades will need to be performed prior to the Lagoon update.
+
 ### Deleted Backups
 * When a backup is deleted via the webhook, it will now actually removed from the API rather than being flagged as deleted. The `Backup` type field `deleted` is deprecated, and will be removed in a future release. Additionally, `includeDeleted` if requested when querying backups will not change the result as there will be no deleted backups to include.
```

Comment:
Woops, missing `be` in this sentence
```
When a backup is deleted via the webhook, it will now actually be removed from the API rather than being flagged as deleted
```